l3backend/l3backend-color.dtx

```diff
@@ -913,6 +913,37 @@
 % \end{macro}
 % \end{macro}
 %
+% \begin{macro}{\@@_backend_iccbased_device:nnn}
+%   This is very similar to setting up a color space: the only part we
+%   skip is adding it to the page resources.
+%    \begin{macrocode}
+\cs_new_protected:Npn \@@_backend_iccbased_device:nnn #1#2#3
+  {
+    \pdf_object_if_exist:nF { @@_icc_ #1 }
+      {
+        \pdf_object_new:nn { @@_icc_ #1 } { fstream }
+        \pdf_object_write:nn { @@_icc_ #1 }
+          {
+            { /N ~ #3 }
+            {#1}
+          }
+      }
+    \pdf_object_unnamed_write:nx { array }
+      { /ICCBased ~ \pdf_object_ref:n { @@_icc_ #1 } }
+    \cs_if_exist:NT \pdfmanagement_add:nnn
+      {
+        \use:x
+          {
+            \pdfmanagement_add:nnn
+              { Page / Resources / ColorSpace }
+              { Default #2 }
```

Comment:
That should be Device not Default

Reply:
The idea is to parallel `colorspace`:
```
/DefaultRGB 0 <X> R
```
where `<X>` is the object ref. for the ICC colour space. Have I got it in the wrong place?

Reply:
I'm not sure ;-) I thought it is Device, but perhaps I mixed it up. I need to check,

Reply:
You are right, it is Default.